qiskit/opflow/gradients/gradient.py

```diff
@@ -108,6 +108,12 @@ def is_coeff_c(coeff, c):
                 return expr == c
             return coeff == c
 
+        def is_coeff_c_abs(coeff, c):
```

Comment:
Can we have type hints and a better name than `c`?

Reply:
I agree it's not great, but we have the same/similar functions in several places in opflow. It might good to keep it the same for internal consistency -- or change them all in a follow up 🙂

Reply:
Ok